src/ride/controllers/create_ride_controller.py

```diff
@@ -118,7 +118,7 @@ def process(self):
             description=description,
             departure_datetime=departure_datetime,
             is_flexible=is_flexible,
-            type=type,
+            type=ride_type,
```

Comment:
Shouldn't we change the type field inside Ride to also be ride_type to avoid the same issue and also keep things symmetric?

Reply:
+1

Reply:
Good point!